governance/votes/Executive vote - June 7, 2023.md

```diff
@@ -0,0 +1,215 @@
+- **RWA014:** `TBD`
+- **MCD_JOIN_RWA015_A:** `TBD`.
+- **RWA015_A_URN:** `TBD`.
+- **RWA015_A_JAR:** `TBD`.
+- **RWA015_A_OUTPUT_CONDUIT:** `TBD`.
+- **RWA015_A_INPUT_CONDUIT_URN:** `TBD`.
+- **RWA015_A_INPUT_CONDUIT_JAR:** `TBD`.
+- **RWA015_A_OPERATOR:** `TBD`.
+- **ETH_Address_A (USDC Wallet In):** `0x65729807485F6f7695AF863d97D62140B7d69d83`.
+
+#### Additional Information
+
+Please review the following forum threads containing information about RWA015 (BlockTower Andromeda) to inform your position before voting:
+- [Initial Collateral Application](https://forum.makerdao.com/t/mip90-liquid-aaa-structured-credit-money-market-fund/18428).
+- [Steakhouse Financial Legal & Risk Assessment](https://forum.makerdao.com/t/project-andromeda-risk-legal-assessment/20969).
+- [dewiz Technical Assessment](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974).
+
+### BlockTower RWA Vault [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmPMrvfV) the following changes to the BlockTower RWA Vault Debt Ceiling parameters will be made, if this executive proposal passes:
+- Decrease the RWA010-A Debt Ceiling by 20 million DAI from 20 million DAI to **0**.
+- Decrease the RWA011-A Debt Ceiling by 30 million DAI from 30 million DAI to **0**.
+- Increase the RWA012-A Debt Ceiling by 50 million DAI from 30 million DAI to **80 million DAI**.
+
+To facilitate these changes, the [RWA Agreement (`doc`)](https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement) for RWA010-A, RWA011-A, RWA012-A, and RWA013-A will be updated to **QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU**, the agreement details may be [viewed here](https://gateway.pinata.cloud/ipfs/QmY382BPa5UQfmpTfi6KhjqQHtqq1fFFg2owBfsD2LKmYU).
+
+### Stability Scope Defined Parameter Changes
+
+As per the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104), the following parameter changes will be made, if this executive proposal passes:
+
+#### [Dai Savings Rate (DSR)](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) Increase
+- Increase the DSR by 2.49% from 1% to **3.49%**.
+
+#### [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee) Increases
+- Increase the ETH-A Stability Fee by 1.99% from 1.75% to **3.74%**.
+- Increase the ETH-B Stability Fee by 0.99% from 3.25% to **4.24%**.
+- Increase the ETH-C Stability Fee by 2.49% from 1% to **3.49%**.
+- Increase the WSTETH-A Stability Fee by 1.99% from 1.75% to **3.74%**.
+- Increase the WSTETH-B Stability Fee by 2.49% from 1% to **3.49%**.
+
+### Non-Scope Defined Paramater Changes
+
+As per [this poll](https://vote.makerdao.com/polling/QmQXhS3Z), the following parameter changes will be made, if this executive proposal passes:
+
+#### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam) Increase
```

Comment:
Personally I'd divide these up differently to reduce the number of headers. I would likely do:

#### rETH Changes
etc

#### Other Stability Fee Changes
etc

Reply:
changed it, see what you think